tools/keytools/sign.c

```diff
@@ -992,7 +1039,7 @@ static int sign_digest(int sign, int hash_algo,
                                    digest, digest_sz, &rng);
         }
         if (ret != 0) {
-            fprintf(stderr, "error signing with XMSS: %d\n", ret);
+            fprintf(stderr, "error signing with ML-DSA: %d\n", ret);
```

Comment:
typo: error signing with XMSS

Reply:
please double check this, it fixes a previous typo/copypaste error. This is actually the message displayed when the ML_DSA signature fails.

Reply:
You're correct, my mistake